Lib/http/server.py

```diff
@@ -1251,6 +1258,42 @@ def run_cgi(self):
                 self.log_message("CGI script exited OK")
 
 
+class HTTPSServer(HTTPServer):
+    def __init__(self, server_address, RequestHandlerClass,
+                 bind_and_activate=True, *, certfile, keyfile=None,
+                 password=None, alpn_protocols=None):
+        if ssl is None:
+            raise ImportError("SSL support missing")
+        if not certfile:
+            raise TypeError("__init__() missing required argument 'certfile'")
```

Comment:
It should be a ValueError. In addition, the argument is not missing. It's just that a falsey argument is invalid. But I think we shouldn't guard this and let exceptions propagte when we access the attribute.